modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -53,7 +53,8 @@ def get_claim
         claim_detail = if Flipper.enabled?(:mobile_lighthouse_claims, @current_user)
                          lighthouse_claims_adapter.parse(lighthouse_claims_proxy.get_claim(params[:id]))
                        else
-                         evss_claims_proxy.get_claim(params[:id])
+                         evss_claim_serializer = evss_claims_proxy.get_claim(params[:id])
```

Comment:
It seems wrong to have two serializers here, but I see that the proxy does indeed return a serializer.